### ISSUE TITLE
Install button stays gray even if the extension is installed

### ISSUE THREAD

**Issue by ValentinaPC** [mozilla/addons-frontend#3441]
Steps to reproduce:
1. Try to install an extension  i.e. https://addons.allizom.org/en-US/firefox/addon/webext/

Expected results:
A permission popup is displayed. Install button turns to green. Add-on is installed.

Actual results:
A permission popup is displayed. Install button remains grey,

Notes/Issues:
- after page refresh install button turns to green.
Logs from FF58 (Win 7) - https://pastebin.com/WGQSmn57

**Comment by muffinresearch** [comment]
@ValentinaPC is this for one add-on or everything? 

@kumar303 would you have a moment to see if you can repro this and diagnose the cause in-case there's something here that needs to be fixed before the push tomorrow?

**Comment by kumar303** [comment]
Whoa, good catch. This seems to be affecting all add-ons. I'm working on a fix right now.

```
[amo] Install error: TypeError: _tracking is undefined
installAddon.js:343:8
```

**Comment by ValentinaPC** [comment]
Verified as fixed on AMO-dev FF58(Win 7, Android 7.2.1).
Postfix video:
![install](https://user-images.githubusercontent.com/15685960/31485279-fa602466-af3b-11e7-8596-3380de839eef.gif)
